test/controllers/sign.out.controller.spec.ts

```diff
@@ -60,8 +60,35 @@ describe("Sign Out controller", () => {
       expect(resp.status).toEqual(200);
       expect(resp.text).toContain(SIGN_OUT_PAGE_TITLE);
       expect(resp.text).toContain('Your answers will not be saved. You will need to start again if you want to register an overseas entity and tell us about its beneficial owners.');
+      expect(resp.text).not.toContain(SIGN_OUT_HELP_DETAILS_TEXT);
+      expect(resp.text).toContain(`${config.REGISTER_AN_OVERSEAS_ENTITY_URL}${config.SOLD_LAND_FILTER_PAGE}`);
+    });
+
+    test(`renders the ${config.SIGN_OUT_PAGE} page, when FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022 is active, and guidance for resuming a saved journey is displayed`, async () => {
+      mockIsActiveFeature.mockReturnValueOnce(true);
+      const resp = await request(app)
+        .get(`${config.SIGN_OUT_URL}?page=${config.SOLD_LAND_FILTER_PAGE}`);
+
+      expect(resp.status).toEqual(200);
+      expect(resp.text).toContain(SIGN_OUT_PAGE_TITLE);
+      expect(resp.text).toContain(SIGN_OUT_HINT_TEXT);
       expect(resp.text).toContain(SIGN_OUT_HELP_DETAILS_TEXT);
+      expect(resp.text).toContain(SIGN_OUT_DROPDOWN_TEXT);
+      expect(resp.text).toContain('How do I find my saved applications?');
```

Comment:
Please define this string with an appropriate constant name (`SIGN_OUT_SAVE_AND_RESUME_GUIDANCE_TEXT`?) in the 'text.mock.ts' file and reference it from there (both here and below). I think you also need to check for the presence of the following string in this test and that it's not present in your other new test below (again, define as a constant, `SIGN_OUT_SAVE_AND_RESUME_GUIDANCE_DETAILS_TEXT` perhaps):

_"Sign in to your Companies House account and select Your filings to access your saved applications."_

Reply:
added